src/pages/docs/creators/ray/outbound.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Accessing the Internet (outbound) 
+description: Discover how to access the internet from your Ray applications with this hands-on guide, featuring code examples, and yaml configurations.
+pageTitle: Practical Tutorial on Accessing the Internet with Ray on Golem
+type: tutorial
+---
+
+# Accessing the Internet (outbound)
+
+The purpose of this article is to show you an example of a Ray app accessing the Internet and to explain the  nuances of running it with Ray on Golem.
+
+## Outbound introduction
+
+Ensuring providers' security is a very important topic within the Golem network. 
+One of the implemented security measures is the ability to restrict the usage of outgoing Internet connections from within the VM images the providers deploy on their computers.
+
+The simplest way to do this is by allowing outgoing connections only to the select few whitelisted addresses.
+The default whitelist is curated by Golem Factory and is kept on [GitHub](https://github.com/golemfactory/ya-installer-resources/tree/main/whitelist).
+
+Providers may edit the whitelist, or disable it altogether to accept all outgoing traffic. 
+In the future, we will add support for allowing outgoing traffic from within images signed by trusted entities (e.g. Golem Factory)
+
+Feel free to check out the general [Accessing the Internet from Golem providers](/docs/creators/javascript/guides/accessing-internet) guide to get more context not related to Ray on Golem.
+
+Please reach out to us on [`#Ray on Golem` discord channel](https://chat.golem.network/) to discuss the options and help us define the best way to support your outbound needs.
+
+## Default whitelist
+
+Please have a look at the [default whitelist](https://github.com/golemfactory/ya-installer-resources/tree/main/whitelist) yourself.
+
+Among the listed entries you can find `dl.dropboxusercontent.com`, `ipfs.io` - they should allow a certain level of freedom when storing artifacts needed by your Ray on Golem applications.
+You can upload the data your computation needs to Dropbox or IPFS and then use it in your remote tasks. This is demonstrated in the [example below](#simple-outbound-example)
+
+One of the ways that Ray on Golem uses the whitelist is to enable installation of [additional packages](/docs/creators/ray/cluster-yaml#initialization-commands) with pip. 
+Golem Factory runs a custom proxy to `pypi.org` via `pypi.dev.golem.network` to make it possible.
+
+When your application requires access to a non-whitelisted URL, the recommended way is to advertise your need on [`#providers` discord channel](https://chat.golem.network/).
```

Comment:
Could it be beneficial to suggest to the requester that they review the documentation available at https://docs.golem.network/docs/providers/configuration/outbound#whitelisting-a-specific-domain? This resource explains how providers can whitelist a domain.

Reply:
good point, addeded the link :heart: